js/constants.js

```diff
@@ -44,13 +46,17 @@
     
     // filter links from string
     root.filterLinks = string => {
-        let parts = string.split('src="http')
-        let result = parts[0]
-        parts.forEach(part => {
-            let index = part.indexOf('"')
-            let newLink = part.slice(0,index).split('/').pop()
-            result += 'src="' + newLink + part.slice(index)
-        })
+        let filteredText = (string || "").replace(root.REGEX_ACTION_LINK, '').replace(root.REGEX_KEEP_URL_PATH_ONLY, '$1')
+        return filteredText
+    }
+
+    root.filterLinksFromNode = node => {
```

Comment:
Note to self and to reviewers:
Replacing the urls directly from `root.extract.question(e)` would cause the `xmlns` attribute to change (since it's an url) and we would've to replace the entire database
This is a workaround and can be discussed